README.md

```diff
@@ -5,30 +5,69 @@
 This repo provides a customizable stack for starting new ML projects
 on Databricks that follow production best-practices out of the box.
 
-The default stack in this repo includes three modular components:
+Using Databricks MLOps Stacks, data scientists can quickly get started iterating on ML code for new projects while ops engineers set up CI/CD and ML service state
```

Comment:
Is it clear what `ML service state management` is?  Terraform state management? It's not a common term, I think?

Reply:
since that is handled by DABs now, we can just say ML asset management to match the other edits